pkg/homedir/homedir.go

```diff
@@ -5,14 +5,20 @@ import (
 	"runtime"
 )
 
+// Return the Env Var Name for the user's home dir. On Linux this is
+// "HOME" while on windows its "USERPROFILE"
```

Comment:
Also I'd call this something else. instead of `homedir.HomeKey()`, `homedir.EnvironmentKey()` seems like a better fit to me.